Sources/Parser/Parser+Declaration.swift

```diff
@@ -174,7 +189,40 @@ extension Parser {
     }
     return EnumMember(caseToken: caseToken, identifier: identifier, type: Type(identifier: enumIdentifier), hiddenValue: hiddenValue, hiddenType: hiddenType)
   }
-  
+
+  func parseTraitMembers() throws -> [TraitMember] {
+    var members = [TraitMember]()
+    while let first = currentToken?.kind {
+      switch first {
+      case .punctuation(.closeBrace):
+        return members
+      default:
+        members.append(try parseTraitMember())
+        //throw raise(.badMember(in: "trait", at: latestSource))
```

Comment:
You probably want: case .punctuation(.at), .public, .visible, .mutating, .func, .event